### ISSUE TITLE
🐛 Bug Report: Horizontal scroll on website works although the scrollbar is hidden

### ISSUE THREAD

**Issue by madhaven** [appwrite/appwrite#7077]
### 👟 Reproduction steps

try scrolling to the right.

### 👍 Expected behavior

it should not show black space

### 👎 Actual Behavior

It shows a black space that i can scroll to.

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Windows

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by Haimantika** [comment]
Hi @madhaven I couldn’t reproduce this, so I need to ask some questions:

1. Do you see it throughout? 
2. Or only on some pages? 
3. Can you reproduce it on mobile?

**Comment by madhaven** [comment]
@Haimantika 
1. I do dee it throughout  
  The only exception is the top area where you have a huge background image (I think) that takes some of that space.  
  FYI I'm using a plugin called Vimium to navigate my page.  
  ![image](https://github.com/appwrite/appwrite/assets/37440750/40bc4758-7f24-4087-aaf8-4968e63bbce5)  
  ![image](https://github.com/appwrite/appwrite/assets/37440750/4f0d04ef-a25d-41d1-92c1-d0e39e5b523c)  
  https://github.com/appwrite/appwrite/assets/37440750/57a33b62-3bf4-4e09-9a44-5161b3a6de5c  
2. I only saw the main page that is linked to this repository, I did not check all of it out.
3. I could not reproduce it on mobile although I did find some other bugs on the mobile
